### DISCUSSION TITLE
custom header

### DISCUSSION THREAD

**Discussion by breakingbad45** [pocketbase/pocketbase#3769] [Q&A]
```
const fetchData = await pb.collection('posts').getList(1, 20, {
      
        }); 
```

i want to send a custom header(like api_key="hghghfrsrserej") with this request.is it possible to set custom header ?

**Comment by j0sh** [comment]
Yes - per the [JS SDK docs](https://github.com/pocketbase/js-sdk#custom-request-options), each request can take a `header` object:

```js
const fetchData = await pb.collection('posts').getList(1, 20, {
  headers: {
    'api_key':'hghghfrsrserej'
  }     
 }); 
```
